test/spec/invertedIndexSpec.js

```diff
@@ -1,27 +1,21 @@
-const validFile = require('./books/validFile.json');
-const simpleValidFile = require('./books/simpleValidFile.json');
+const books = require('./books/validFile.json');
+const shortStories = require('./books/simpleValidFile.json');
```

Comment:
A base filename should exactly match the name of its default export. Please refer to your [Airbnb Style guide](https://github.com/airbnb/javascript#naming--filename-matches-export)